v19.2/unsplit-at.md

```diff
@@ -0,0 +1,192 @@
+---
+title: UNSPLIT AT
+summary: The UNSPLIT AT statement reverts a manual key-value layer range split at a specified row in a table or index.
+toc: true
+---
+
+The `UNSPLIT AT` [statement](sql-statements.html) removes a [split enforcement](split-at.html) on a key-value layer range [split](architecture/distribution-layer.html#range-splits), at a specified row in a table or index.
```

Comment:
I would make "range split" the link text.  It would be nice to have another sentence or two that describes what happens when the "split enforcement" is removed, even if it's just something simple like "this allows CRDB to merge the ranges back together at its leisure" or similar (but better :-} ).  Additionally, is there some cluster setting that would control how slowly or quickly this merging will happen?  If so I think we should mention that and add linkage.  Also, I think we should add a callout that links to the [Range Merges doc](https://www.cockroachlabs.com/docs/v19.1/range-merges.html) and mentions that the range merge cluster setting.  Finally I think we should make the behavior of how this SQL statement interacts with that setting clear, e.g., "for the UNSPLIT AT statement's removal of the split enforcement to result in ranges being merged back together at the specified point, the kv.range_merge.queue_enabled setting must be set to true"  (If that sentence is correct, I have not verified/tested that)